setup.cfg

```diff
@@ -72,5 +75,5 @@ max-line-length = 79
 max-complexity = 18
 exclude = __init__.py|examples
 per-file-ignores =
-    # imported but unused
-    __init__.py: F401
+  # imported but unused
+  __init__.py: F401
```

Comment:
Just had a check, and we don't violate `F401` at present, so we could probably get rid of `per-file-ignores` altogether